src/components/InCall.tsx

```diff
@@ -79,5 +102,17 @@ export const InCall = ({
     return null;
   }
 
-  return <div ref={divRef} css={{ height: "100%" }} />;
+  const hidden = {
+    opacity: "0",
+    pointerEvents: "none" as const,
+    position: "fixed" as const,
+    zIndex: -1,
+  };
+
+  return (
+    <>
+      <div ref={divRef} css={{ height: "100%" }} />{" "}
+      <main style={hidden}>{transcript}</main>{" "}
```

Comment:
nit: use `css=` here rather than style, as that will cause emotion to generate a css class for this rather than just inline stye. It would also avoid the need for the `as const` above if you wrapped the declaration in a `css` constructor. (You could also achieve that by just by inlining `hidden`.)

Not critical in this case though as the style is only used once, so the generation of named style classes by emotion won't have any performance improvement.